src/main/kotlin/io/snyk/plugin/ui/toolwindow/VulnerabilityDescriptionPanel.kt

```diff
@@ -324,6 +325,13 @@ class VulnerabilityDescriptionPanel(private val vulnerability: Vulnerability) :
                 0,
                 false))
 
+        // don't remove that!
+        // Some magic (side-effect?) happens when JBHtmlEditorKit() initializing
+        // that make html tags like <em>, <p>, <ul> etc. be treated properly inside JEditorPane
+        // PS occasionally found that when played with com.intellij.util.ui.HtmlPanel
```

Comment:
Don't sure about such comments in master 😉. But it conveys a thought well

Reply:
At least it takes your attention and (and will for me as well few weeks/month later), that was the mean reason for such comment